ui/src/gear/permissions/types.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ *  A server identity
+ */
+type Dude = string;
+
+/**
+ * path - ship desk case spur
+ */
+type Spur = string;
+
+/**
+ * TODO
```

Comment:
`burr` and `spar` are defined thusly:

```hoon
  +$  burr  [desk=(unit desk) =spur]
  +$  spar  [care=(unit term) burr]
```

What is a good label for them in this type description?

Reply:
Something like:

burr: specific or any desk, at or under spur prefix
spar: burr with specific or any care